backend/src/zimfarm_backend/db/tasks.py

```diff
@@ -41,28 +45,33 @@ class TaskListResult(BaseModel):
     tasks: list[TaskLightSchema]
 
 
+def create_task_file_schema(file: File) -> TaskFileSchema:
+    return TaskFileSchema(
+        name=file.name,
+        size=file.size,
+        status=file.status,
+        created_timestamp=file.created_timestamp,
+        uploaded_timestamp=file.uploaded_timestamp,
+        failed_timestamp=file.failed_timestamp,
+        check_timestamp=file.check_timestamp,
+        check_result=file.check_result,
+        check_log=file.check_log,
+        check_details=file.check_details,
+        info=file.info,
+        cms=CMSStatusSchema(
```

Comment:
Let's unnest these properties, it is not like they are really used by any of our external systems ATM, only our own UI, so better to do it now than regret it later 🤣 (and you can then change comment in backend/src/zimfarm_backend/db/models.py)